packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -178,6 +180,10 @@ class MarkdownBuilder implements md.NodeVisitor {
   String? _lastVisitedTag;
   bool _isInBlockquote = false;
 
+  bool _isBlockTag(String? tag) =>
+      _kBlockTags.contains(tag) ||
+      (customBlockTags ?? <String>[]).contains(tag);
```

Comment:
This looks like the answer to the previous question is that it should be non-nullable.